### PR TITLE
AT_02.04.004 | Homepage(Dashboard) | Verify "New Item" redirection

### PR DESCRIPTION
**Pull request by ellayezina**
https://trello.com/c/dFPzqj8R/799-at0204004-homepagedashboard-verify-new-item-redirection

### CODE REVIEW

File: cypress/e2e/dashboardMenu.cy.js [hunk 1]
```diff
@@ -15,4 +15,9 @@ describe('Homepage(Dashboard)| Side panel sub-menu', () => {
             expect($el.text()).to.be.equal(sidePanelItems[idx])
         })
     })
+
+    it('Homepage(Dashboard) | Verify "New Item" redirection', () => {
```
Comment (IrynaKolh) [inline cypress/e2e/dashboardMenu.cy.js:19]: вы пропустили № АТ

**Comment by milladiz84** [comment]
Could you please add the GitHub pull request link to your automation card in Trello.

**Review by milladiz84 — COMMENTED** [review]
Could you please add the GitHub pull request link to your automation card in Trello.

File: cypress/e2e/dashboardMenu.cy.js (hunk 1 above)
Comment (ellayezina) [reply cypress/e2e/dashboardMenu.cy.js:19]: Will fix. Thanks

**Review by IrynaKolyada — APPROVED** [review]

**Pull request merged**